pkgs/desktops/gnome/core/gnome-color-manager/default.nix

```diff
@@ -37,12 +37,12 @@ stdenv.mkDerivation rec {
     ninja
     pkg-config
     gettext
+    glib
     itstool
     desktop-file-utils
   ];
 
   buildInputs = [
-    glib
```

Comment:
It should still remain here as it is used as a library dependency. GTK probably won't lose glib propagation but it is better to keep it explicit.